8_reference/74_package_format.txt

```diff
@@ -0,0 +1,78 @@
+=== Package format
+
+Rudder has a specific package format for plugins.
+
+You can manage Rudder packages with the rudder-pkg command. This is the documentation of how they are created.
+
+==== File description
+
+A Rudder package file ends with the .rpkg extension.
+
+A Rudder package file is an archive file and can be managed with the 'ar' command.
+
+The archive contains:
+ * A metadata file in JSON format named medatata
+ * A tarball file in txz format name scripts.txz that contains package setup utility scripts
+ * One or more tarball files in txz format that contain the package files
+
+The metadata file is a JSON file and is named 'metadata':
+```
+{
+  # the only currently supported type in "plugin" (mandatory)
+  "type": "plugin",
+  # the plugin name must consist of ascii characters without whitespace (mandatory)
+  "name": "myplugin",
+  # the plugin version has the form "rudder_major-version_major.version_minor" (mandatory)
+  "version": "4.1-1.0",
+  # this is purely informative (optional)
+  "build-date": "2017-01-20",
+  # the list of jar files to enable if this is a webapp plugin (optional)
+  "jar-files": [ "test.jar" ],
+  # the list of packages or other plugins that this package depends on (optional)
+  # this is currently only informative
+  "depends": {
+    # dependencies on dpkg based systems
+    "dpkg": [ "apache2" ],
+    "rpm": [ ],
+    # dependency specific to debian-8
+    "debian-8": [ ],
+    "sles-11": [ ],
+    # rudder dependency, ie this is a Rudder format package
+    "rudder": [ "new-plugin" ]
+  },
```

Comment:
This means removing the example plugin will lead to delete /opt/rudder/share and /var/rudder?